### DISCUSSION TITLE
Pull requests can’t load

### DISCUSSION THREAD

**Discussion by kddnewton** [community/community#69565] [Mobile]
I cannot load the pull requests view when I have an open PR on an organization that I am not signed in to.

For more information: I have pull requests submitted to open source repos and to my work repos. I would like to be able to see my open source repos when I click on pull requests. But if I left a work PR open, the page just crashes (presumably because I'm not actually allowed to see those since I'm not signed into that org on my phone). But this renders the whole page unusable. Ideally I would just see the PRs I'm allowed to see based on which orgs I'm signed in to.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.133.1 (112099449)
Device: iPhone 12 (iOS 16.3.1)
</details>

**Comment by elisealix22** [comment]
Hi @kddnewton , thank you so much for the detailed report and I'm sorry you're running into this issue. I opened an internal bug report for our iOS team to investigate further. Let us know if you have any other requests in the meantime.

**Comment by HenSquared** [comment]
Hi @kddnewton and anyone else facing this issue,

If you clear cache, does the issue still occur?

To clear cache you can:
Go to profile tab
Tap settings icon (Top right corner of profile screen)
Tap 'clear cache' (bottom of settings screen)

I'm currently not able to reproduce the issue, so if you have other details that may be helpful, please share. Thanks!

**Comment by kddnewton** [comment]
Hi @HenSquared - sorry clearing the cache still appears to not work.

I managed to close my last work PR and that fixed it. Then I opened another one to test and it's broken again.

**Reply by HenSquared** [reply]
Thank you for trying that and for the details!. We will continue investigation on this
